src/components/Footer/index.tsx

```diff
@@ -1,146 +1,91 @@
-          <polygon
-            className="text-gray-800 fill-current"
-            points="2560 0 2560 100 0 100"
-          ></polygon>
-        </svg>
-      </div>
-      <div className="container mx-auto px-4 p-8 pb-6">
-        <div className="flex flex-wrap">
-          <div className="w-full lg:w-6/12 px-4 text-center md:text-left">
-            <h4 className="text-2xl font-semibold text-white">
-              Buscanos en nuestras redes
-            </h4>
-            <div className="flex space-x-1 mt-6 justify-center md:justify-start">
-              <Link href="https://twitter.com/FrontEndCafe">
-                <a
-                  target="_blank"
-                  className="grid place-items-center h-10 w-10 rounded-full bg-white text-blue-500 hover:bg-gray-200"
-                >
-                  <FontAwesomeIcon icon={faTwitter} size="lg" />
-                </a>
-              </Link>
-
-              <Link href="http://github.com/frontend-cafe">
-                <a
-                  target="_blank"
-                  className="grid place-items-center h-10 w-10 rounded-full bg-white text-gray-900 hover:bg-gray-200"
-                >
-                  <FontAwesomeIcon icon={faGithubAlt} size="lg" />
-                </a>
-              </Link>
+   const navItems = [
+    { title: 'Unirse a Discord', link: 'https://discord.com/invite/3GC6TJd' },
+    { title: 'Proyectos CMYK', link: '/cmyk' },
+    { title: 'Mentorías', link: '/mentorias' },
+    { title: 'Prácticas de Ingles', link: '/ingles' },
+  ];
 
-              <Link href="https://www.youtube.com/channel/UCUdXQMrVjrMMWG5NOZFpHqQ">
-                <a
-                  target="_blank"
-                  className="grid place-items-center h-10 w-10 rounded-full bg-white text-red-700 hover:bg-gray-200"
-                >
-                  <FontAwesomeIcon icon={faYoutube} size="lg" />
-                </a>
-              </Link>
-
-              <Link href="https://www.linkedin.com/company/frontendcafe/">
-                <a
-                  target="_blank"
-                  className="grid place-items-center h-10 w-10 rounded-full bg-white text-blue-700 hover:bg-gray-200"
-                >
-                  <FontAwesomeIcon icon={faLinkedin} size="lg" />
-                </a>
-              </Link>
-
-              <Link href="https://www.twitch.tv/frontendcafe">
-                <a
-                  target="_blank"
-                  className="grid place-items-center h-10 w-10 rounded-full bg-white text-purple-700 hover:bg-gray-200"
-                >
-                  <FontAwesomeIcon icon={faTwitch} size="lg" />
-                </a>
-              </Link>
-            </div>
+  return (
+    <footer className="relative md:h-64 h-80 bg-black lg:px-32 md:px-20 md:py-8 py-5 text-white lg:text-xl text-md flex flex-col justify-between items-center md:items-start">
+      <img src='/logo-square.png' className='md:h-12 md:w-12 h-10 w-10 rounded-full'></img>
```

Comment:
We are mixing the use of `"` with `'` in here, which is typically undesired. I would recommend you to install [Prettier](https://prettier.io/) which will automatically change them for you no matter what you use.

Reply:
I will do it by hand, something is wrong with my prettier config